database/connection.ts

```diff
@@ -0,0 +1,8 @@
+import mongoose from "mongoose";
+
+export const main = async () => {
+  mongoose.connect(
+    "mongodb+srv://admin:UhBnjYwCunPstXes@github-profile-viewer.uaxxp8r.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
please put sensitive info like passwords and tokens into a .env file and not directly into the code for everyone to see 😛 also best you change that password now since its already exposed to the public